components/InformationTile.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+    <div class="rectangle-info"><p class="tile-content">{{ textContent }}</p></div>
+</template>
+
+<script lang="ts">
+export default {
```

Comment:
Since we moved to Vue 3, we want to start using Composition API and TypeScript. So this should be <script setup lang="ts">. This means you'll have to define props with defineProps instead.

See our Vue workshop or [vuejs.org/guide/components/props.html](https://vuejs.org/guide/components/props.html) for examples.